uk.co.compucorp.civicrm.tasksassignments/js/src/tasks-assignments/controllers/modal/modal-document.js

```diff
@@ -37,6 +37,26 @@ define([
         form: false
       };
 
+      (function init () {
```

Comment:
I didn't mean just moving the init. 

![image](https://user-images.githubusercontent.com/3973243/28928010-fddba75a-7863-11e7-8c3e-cf40bac44ec1.png)
Styleguide ^^

It is as well about making functions private and then bonding them with `vm`